tools/run_tests/filter_pull_request_tests.py

```diff
@@ -174,8 +175,8 @@ def filter_tests(tests, base_branch):
   print("Finding file differences between gRPC %s branch and pull request...\n" % base_branch)
   changed_files = _get_changed_files(base_branch)
   for changed_file in changed_files:
-    print(changed_file)
-  print
+    print("  %s" % changed_file)
```

Comment:
nit: use consistent quote type. Our other python scripts use mostly `'`, so let's get rid of the double quotes.